engines/docker/engine_test.go

```diff
@@ -4,8 +4,15 @@ import (
 	"os"
 	"testing"
 
-	"github.com/taskcluster/taskcluster-worker/engines/enginetest"
 	"time"
+
+	"github.com/taskcluster/taskcluster-worker/engines/enginetest"
+)
+
+// Image and tag used in test cases below
+const (
+	dockerImageRepository = "alpine"
+	dockerImageTag        = "3.6"
```

Comment:
The image hash disappeared from the repository without warning... So I guess maybe we have to use tags...
Or well, at-least we might have to support both.